api/hooks/customGameHook/index.js

```diff
@@ -3,11 +3,13 @@ module.exports = function gameHook() {
   // Game API //
   //////////////
   return {
-    createGame: function (gameName) {
+    createGame: function (gameName, isRanked = false) {
```

Comment:
I asked about this earlier so it's good to see it has a logical default for `isRanked`.